data-reconciliation-app/docker/ccf_app_js.virtual

```diff
@@ -0,0 +1,20 @@
+# Build
+FROM mcr.microsoft.com/ccf/app/dev:2.0.7-sgx as builder
+
+# Run
+FROM mcr.microsoft.com/ccf/app/run-js:2.0.7-sgx
+
+# Note: libjs_generic.virtual is not included in run-js container
+COPY --from=builder /opt/ccf/lib/libjs_generic.virtual.so /usr/lib/ccf
+
+# copy configuration into image
+COPY ./governance/constitution/*.js /app/
+COPY ./workspace/docker_certificates/cchost_config_virtual_js.json /app/
```

Comment:
adding a comment here to point out that both virtual and enclave files should be changed